packages/contracts-bedrock/scripts/deploy/DeployOPChain.s.sol

```diff
@@ -579,7 +579,7 @@ contract DeployOPChain is Script {
 
         require(address(messenger.PORTAL()) == address(_doo.optimismPortalProxy()), "L1xDM-30");
         require(address(messenger.portal()) == address(_doo.optimismPortalProxy()), "L1xDM-40");
-        require(address(messenger.superchainConfig()) == address(_doi.opcm().superchainConfig()), "L1xDM-50");
```

Comment:
Now that we've made this change, we need to add an assertion into `assertValidSystemConfig` that checks that the SuperchainConfig is configured correctly.